src/main/java/cope/cosmos/client/ui/altmanager/AltEntry.java

```diff
@@ -0,0 +1,142 @@
+package cope.cosmos.client.ui.altmanager;
+
+import cope.cosmos.asm.mixins.accessor.IMinecraft;
+import cope.cosmos.client.ui.util.InterfaceUtil;
+import cope.cosmos.util.render.FontUtil;
+import cope.cosmos.util.render.RenderUtil;
+import cope.cosmos.util.string.ColorUtil;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.Gui;
+import net.minecraft.client.gui.ScaledResolution;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.TextFormatting;
+import org.lwjgl.opengl.GL11;
+
+/**
+ * @author Wolfsurge
+ */
+public class AltEntry implements InterfaceUtil {
+
+    // The alt
+    private Alt alt;
+    // The offset of the button
+    private float offset;
+    // Is this the selected alt
+    public boolean isSelected;
+
+    public AltEntry(Alt alt, float offset) {
+        this.alt = alt;
+        this.offset = offset;
+    }
+
+    /**
+     * Draws the alt entry
+     * @param mouseX The mouse's X
+     * @param mouseY The mouse's Y
+     */
+    public void drawAltEntry(int mouseX, int mouseY) {
+        // Gets the scaled resolution
+        ScaledResolution scaledResolution = new ScaledResolution(Minecraft.getMinecraft());
+
+        // Background
+        RenderUtil.drawRect((scaledResolution.getScaledWidth() / 2f) - 150, getOffset(), 300, 30, 0x95000000);
+
+        // Draw border if the mouse is over the button
+        if(isMouseOverButton(mouseX, mouseY))
+            RenderUtil.drawBorder((scaledResolution.getScaledWidth() / 2f) - 150, getOffset(), 300, 30, ColorUtil.getPrimaryColor().darker().darker());
+
+        // Selected outline
+        if(isSelected)
+            RenderUtil.drawBorder((scaledResolution.getScaledWidth() / 2f) - 150, getOffset(), 300, 30, ColorUtil.getPrimaryColor());
+
+        // Rotate and draw arrow. Could do with just having it pre-rotated, but I don't want to mess anything else up.
+        if(getAlt().getAltSession() != null) {
+            GL11.glPushMatrix();
```

Comment:
use static imports (instead of GL11.glPushMatrix() use glPushMatrix() by using a wildcard on the GL11 class)